diff --git a/node_modules/expo-router/build/native-tabs/NativeTabsView.js b/node_modules/expo-router/build/native-tabs/NativeTabsView.js
index 14822ae..e23cf2d 100644
--- a/node_modules/expo-router/build/native-tabs/NativeTabsView.js
+++ b/node_modules/expo-router/build/native-tabs/NativeTabsView.js
@@ -84,6 +84,7 @@ function NativeTabsView(props) {
         }
         : undefined;
     return (<TabsHostWrapper 
+    nativeContainerStyle={props.nativeContainerStyle}
     // #region android props
     tabBarItemTitleFontColor={currentTabAppearance?.stacked?.normal?.tabBarItemTitleFontColor ??
             androidMaterialDefaults?.inactiveColor} tabBarItemTitleFontFamily={currentTabAppearance?.stacked?.normal?.tabBarItemTitleFontFamily} tabBarItemTitleFontSize={currentTabAppearance?.stacked?.normal?.tabBarItemTitleFontSize} tabBarItemTitleFontSizeActive={currentTabAppearance?.stacked?.normal?.tabBarItemTitleFontSize} tabBarItemTitleFontWeight={currentTabAppearance?.stacked?.normal?.tabBarItemTitleFontWeight} tabBarItemTitleFontStyle={currentTabAppearance?.stacked?.normal?.tabBarItemTitleFontStyle} tabBarItemIconColor={currentTabAppearance?.stacked?.normal?.tabBarItemIconColor ??
